RetailCoder.VBE/UI/FindSymbol/FindSymbolDialog.cs

```diff
@@ -26,6 +26,8 @@ private void viewModel_Navigate(object sender, NavigateCodeEventArgs e)
         public FindSymbolDialog()
         {
             InitializeComponent();
+
+            this.Text = string.Format("Rubberduck - {0}", RubberduckUI.FindSymbolDialog_Caption);
```

Comment:
As I understood from the Translation I did, the "Rubberduck - "-Part usually is included in the localized String. (I may be wrong though)